docroot/themes/humsci/humsci_basic/src/scss/partials/ckeditor/_imports.scss

```diff
@@ -363,3 +363,12 @@
 .paragraph-type--stanford-gallery .hs-paragraphs-tabs.tabs {
   display: none;
 }
+
+// Styles for help text in component field in private page
+[data-drupal-selector="edit-field-hs-priv-page-components"] {
```

Comment:
This is not the right place to add this, this stylesheet is only for styles _inside_ the ckeditor widget. Please add it to the [admin theme CSS](https://github.com/SU-HSDO/suhumsci/blob/c58ee5a8e4ea08e121caf205cc999f947773c987/docroot/themes/humsci/su_humsci_gin_admin/dist/su_humsci_gin_admin.css)